src/functions.jl

```diff
@@ -6,8 +6,8 @@ Returns a DataFrame containing function values for binned variables of `df`.
 
 # Arguments
 - `axis_col`: binning axes column(s)
-- `axis_edges`: bin edges for `axis_col`s, as vector-like things or ranges.
-- `bin_col`: column variable(s) to be binned on.  multiple columns essentially form a multidimensional array.
```

Comment:
binstats only returns vector outputs... a multidimensional array can be constructed from the output, but this is a post-processing step